### PR TITLE
Add 2.2 blog post

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Can anyone please review English?

- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Preview](https://deploy-preview-9589--prettier.netlify.app/blog/2020/11/20/2.2.0.html)✨**

### CODE REVIEW

**Comment by fisker** [comment]
Please mention #8937 in the blog post, and we'll merge that `.pre-commit-hooks.yaml` file after release.

Something like, 

````markdown
`pre-commit` support has moved to the new repository https://github.com/prettier/pre-commit, please update your `.pre-commit-config.yaml` file 

```diff
- - repo: https://github.com/prettier/prettier
+ - repo: https://github.com/prettier/pre-commit
-   rev: "2.2.0"
+   rev: "v2.2.0"
   hooks:
    - id: prettier
```
````

**Comment by fisker** [comment]
Need revert #9209

**Comment by sosukesuzuki** [comment]
Is #9209 reverted at #9500?

File: website/blog/2020-11-04-2.2.0.md [hunk 1]
```diff
@@ -0,0 +1,865 @@
...
+// Prettier 2.1
+foo/*:: <bar> */(baz);
+class Foo {
+  bar(data: Array<string>) {}
+}
+```
+
+### CSS
+
+#### Improve custom properties format ([#9209](https://github.com/prettier/prettier/pull/9209) by [@fisker](https://github.com/fisker))
```
Comment (fisker) [inline website/blog/2020-11-04-2.2.0.md:526]: So, just changelog didn't get reverted?

Comment (sosukesuzuki) [reply website/blog/2020-11-04-2.2.0.md:526]: Oh, this is my mistake. I'll fix.

Comment (fisker) [reply website/blog/2020-11-04-2.2.0.md]: https://github.com/prettier/prettier/pull/9589/commits/2527e8041ce32d94eb14344cb9d71b3031ca1102#diff-b3b191bef4c95bbbb93aeae7a119d6cf4e39e6d1710e4a956dd71b46503189abL836

File: website/blog/2020-11-04-2.2.0.md [hunk 2]
```diff
@@ -0,0 +1,820 @@
...
+
+A new value for the `parser` option has been added: [`espree`](https://github.com/eslint/espree) - which is the [`ESLint`](https://github.com/eslint/eslint) default parser.
+
+Note that `espree` only works for [ECMAScript Finished Proposals](https://github.com/tc39/proposals/blob/master/finished-proposals.md), and is stricter than the `babel` parser.
+
+### TypeScript
+
+#### Support TypeScript 4.1 via Babel ([#9473](https://github.com/prettier/prettier/pull/9473) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+`--parser=babel-ts` is required.
```
Comment (kachkaev) [inline website/blog/2020-11-04-2.2.0.md:25]: Maybe worth including a sentence on the future plans? As a Prettier+TS user who is not rushing to try 4.1 features, should I do anything with my projects’ configs?

**Comment by thorn0** [comment]
Remember not to edit the generated file directly.

File: website/blog/2020-11-04-2.2.0.md (hunk 2 above)
Comment (sosukesuzuki) [reply website/blog/2020-11-04-2.2.0.md:25]: I tried to improve the changelog. WDYT? https://github.com/prettier/prettier/pull/9589/commits/5091e02eafe2c37c1422ccae2d7d6184e4930505
/cc @fisker @thorn0

Comment (kachkaev) [reply website/blog/2020-11-04-2.2.0.md:25]: That's clearly better now! One extra thing I would add is a sentence on the expected longevity of this override. As a developer who is planning to use TS 4.1, 4.2 and so on in the future, do I have to keep the override in my config file from now on and forever? Or is it just some temporary thing until, say, Prettier v2.3 is released, after which I can remove the hack?

Personally, I’m not rushing to use TS 4.1 features, but I might need them in a couple of months. The practical question I have is if I should bother updating my shared prettier config with this override or just wait for the next release of Prettier.

Comment (sosukesuzuki) [reply website/blog/2020-11-04-2.2.0.md:25]: Thank you! That's certainly important. I've fixed 09f0c22

File: changelog_unreleased/blog-post-intro.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-author: "🚧"
-authorURL: "https://github.com/🚧"
-title: "Prettier 🚧"
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.2: new JavaScript parsers, TS 4.1 and ESM standalone bundles"
 ---
 
-🚧 Write an introduction here.
+This release supports new JavaScript parsers [espree](https://github.com/eslint/espree) and [meriyah](https://github.com/meriyah/meriyah), supports [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-rc/), ships ESM standalone bundles for modern browsers, and includes many bug fixes and improvements!
```
Comment (fisker) [inline changelog_unreleased/blog-post-intro.md:7]: Shouldn't this file unchanged?

Comment (sosukesuzuki) [reply changelog_unreleased/blog-post-intro.md:7]: This file is used to generate a draft (https://github.com/prettier/prettier/blob/master/scripts/draft-blog-post.js#L81). Also this was used for 2.0 blog (https://github.com/prettier/prettier/pull/7417/files#diff-15f290566c72ae2b4a5ef5b40d9850ddbeeb55e260db4e0a653332d8f603cb6a).

Comment (fisker) [reply changelog_unreleased/blog-post-intro.md:7]: I know that, but isn't this file a template? Or will it restore at release?

Comment (sosukesuzuki) [reply changelog_unreleased/blog-post-intro.md:7]: I'll fix this file when I clear changelog_unreleased, just as when 2.0 is released (https://github.com/prettier/prettier/commit/8f7164fdf49c8241ffaca2f1c7cc4a3f6af06e2a#diff-15f290566c72ae2b4a5ef5b40d9850ddbeeb55e260db4e0a653332d8f603cb6a).

**Review by fisker — APPROVED** [review]

File: website/blog/2020-11-19-2.2.0.md [hunk 4]
```diff
@@ -0,0 +1,1135 @@
...
+title: "Prettier 2.2: new JavaScript parsers, TS 4.1 and ESM standalone bundles"
+---
+
+This release supports new JavaScript parsers [espree](https://github.com/eslint/espree) and [meriyah](https://github.com/meriyah/meriyah), supports [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-rc/), ships ESM standalone bundles for modern browsers, and includes many bug fixes and improvements!
+
+<!--truncate-->
+
+## Highlights
+
+### JavaScript
```
Comment (fisker) [inline website/blog/2020-11-19-2.2.0.md:13]: I think we can remove categories for Highlights?

**Review by Skalman — COMMENTED** [review]
A big thanks to the maintainers! Prettier remains one of my favorite tools. What did we ever do before Prettier? :smile:

File: website/blog/2020-11-19-2.2.0.md (hunk 4 above)
Comment (sosukesuzuki) [reply website/blog/2020-11-19-2.2.0.md:13]: I prefer current structure.

**Comment by sosukesuzuki** [comment]
@Skalman Thank you! I've fixed at c99d274 !

**Pull request merged**